api/src/opentrons/hardware_control/ot3api.py

```diff
@@ -1295,6 +1295,16 @@ async def _retrieve_home_position() -> Tuple[
             await self._backend.home([axis], self.gantry_load)
         else:
             # both stepper and encoder positions are invalid, must home
+            if axis in Axis.pipette_axes():
+                # if we're homing a pipette for the first time, should do it
+                # at the homing current
+                mount = Axis.to_ot3_mount(axis)
+                instr = self._pipette_handler.hardware_instruments[mount]
+
+                if instr is not None:
+                    await self._backend.set_active_current(
```

Comment:
```suggestion
async with self._backend.restore_current():
                    await self._backend.set_active_current(
```

(The home will also need to be wrapped in this)